### PR TITLE
[GH-#240] Readme now more new user readable

### PR DESCRIPTION
**Pull request by nedelcho-delchev-tues**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,51 +109,67 @@ To suspend/stop the miner from mining:
...
+  `GET localhost:4000/tx/{account_hash}`
+
+  Where *account_hash* is the hash representation of an account binary
 
 - To get a block by hash
 
-  `localhost:4000/block/"block_hash"`
+  `GET localhost:4000/block/{block_hash}`
+
+  Where *block_hash* is the bech32 representation of a block structure
```
Comment (d-velev) [inline README.md]: should be where block_hash is a bech32 encoded block header hash or something like that

File: README.md [hunk 2]
```diff
@@ -109,51 +109,67 @@ To suspend/stop the miner from mining:
...
 
 - To get an account balance:
 
-  `localhost:4000/balance/"account_hash"`
+  `GET localhost:4000/balance/{account_hash}`
+
+  Where *account_hash* is the hash representation of an account binary
```
Comment (d-velev) [inline README.md]: the parameter is account not account_hash, account is a hex encoded public key right now, will be bech32 in the future

File: README.md [hunk 3]
```diff
@@ -109,51 +109,67 @@ To suspend/stop the miner from mining:
...
 
 - To get the transaction pool of an account:
 
-    `localhost:4000/tx_pool/"account_hash"`  
+  `GET localhost:4000/tx_pool/{account_hash}`  
+
+  Where *account_hash* is the hash representation of an account binary
```
Comment (d-velev) [inline README.md]: same as the comment above

File: README.md [hunk 4]
```diff
@@ -109,51 +109,67 @@ To suspend/stop the miner from mining:
...
+  Body: **serialized_block**
+
+  Where *serialized_block* is a json serialized block structure
 
 - To get all transactions for an account
 
-  `localhost:4000/tx/"account_hash"`
+  `GET localhost:4000/tx/{account_hash}`
+
+  Where *account_hash* is the hash representation of an account binary
```
Comment (d-velev) [inline README.md]: same as the comment below

**Comment by thepiwo** [comment]
@d-velev can you approve now?

**Pull request merged**